### PR TITLE
Expand the README

### PR DESCRIPTION
**Pull request by strugee**
In particular, add a note that setup instructions are in the app itself, since they're kind of deemphasized in the UI. This confused me for a moment when I didn't notice them, went looking for instructions in the README instead, and found nothing.

Installation instructions shamelessly cribbed from nextcloud/photos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
 # Google integration in Nextcloud
 
 🇬 Google integration allows you to automatically import your Google calendars, contacts, photos and files into Nextcloud.
+
+## 🚀 Installation
+
+In your Nextcloud, simply enable the Google Integration app through the Apps management. The Google Integration app is only included in Nextcloud v20 and higher.
```
Comment (eneiluj) [inline README.md:7]:
```suggestion
In your Nextcloud, simply enable the Google Integration app through the Apps management. The Google Integration app is only included in Nextcloud 20 and higher.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+
+## 🔧 Setup
+
+The app needs some setup in the Google API Console in order to work. To do this, go to Settings > Administration > Connected accounts and follow the instructions under "Google integration".
```
Comment (eneiluj) [inline README.md:11]: It's not clear that your first talk about the google side and then the Nextcloud settings.

```suggestion
The app needs some setup in the Google API Console in order to work. To do this, go to Nextcloud Settings > Administration > Connected accounts and follow the instructions of "Google integration" section.
```

**Review by eneiluj — CHANGES_REQUESTED** [review]
Yay thanks! A few remarks.

**Comment by julien-nc** [comment]
Manually merged with small adjustments.

**Comment by strugee** [comment]
Ack, sorry I never got around to followup work here. Thank you :-)

**Pull request closed without merging**